bus/bus.go

```diff
@@ -1342,9 +1347,10 @@ func (b *bus) Handler() http.Handler {
 		"PUT    /objects/*path": b.objectsHandlerPUT,
 		"DELETE /objects/*path": b.objectsHandlerDELETE,
 
-		"POST   /slabs/migration": b.slabsMigrationHandlerPOST,
-		"GET    /slab/:key":       b.slabHandlerGET,
-		"PUT    /slab":            b.slabHandlerPUT,
+		"POST   /slabs/migration":       b.slabsMigrationHandlerPOST,
+		"POST   /slabs/recomputehealth": b.slabsRecomputeHealthHandlerPOST,
```

Comment:
`nit` (wasn't going to initially but since we're exposing it...)
- `refreshhealth` ?
- `recalculatehealth` ?

I'm not against `recomputehealth` so feel free to resolve this comment if you feel most comfortable with the term 'recompute'.